subprojects/docs/src/samples/build-organization/cross-project-output-sharing/groovy/consumer/build.gradle

```diff
@@ -0,0 +1,18 @@
+configurations {
+    sharedConfiguration {
+        canBeConsumed = false
+        canBeResolved = true
+    }
+}
+
+dependencies {
+    sharedConfiguration(project("path": ":producer", "configuration": "sharedConfiguration"))
+}
+
+tasks.register("showFile") {
+    def sharedConfiguration = configurations.getByName("sharedConfiguration")
+    it.inputs.files(sharedConfiguration)
+    it.doFirst {
+        logger.lifecycle("File is at {}", sharedConfiguration.singleFile.absolutePath)
```

Comment:
Also for this one, does it work with configuration cache? It would be good that all new samples work out of the box with it. I am not sure about this case, since it uses configuration directly

Reply:
The Groovy is compatible with the configuration cache, although I'm not sure if that's really intended. The use of `Configuration` inside the `doFirst{}` gets flagged as incompatible in the Kotlin build script, but for Groovy it's ignored.